src/core/signature.zig

```diff
@@ -28,19 +28,25 @@ pub const Signature = struct {
         return .{ .data = try base58.decode(str) };
     }
 
-    pub fn verify(self: Self, pubkey: Pubkey, msg: []const u8) bool {
+    pub fn verify(
+        self: Self,
+        pubkey: Pubkey,
+        msg: []const u8,
+    ) e.NonCanonicalError!bool {
```

Comment:
Should there be a consensus on when to use horizontal/vertical formatting? I have seen instances where horizontal formatting has been updated to vertical formatting (and vice versa) in some PRs.

Reply:
I format it vertically when the line takes up more than ~100 characters since it goes off my screen.
We could maybe come to a maximum number of characters per row as part of our style guide?

Reply:
I also aim for 100 characters and would like it in the style guide. I have a slight hesitation because there are some exceptions to this rule, but they are very rare.

Reply:
I agree. Could we check this in CI along with unused imports?

Reply:
I would not be opposed to that.

Reply:
IMO we should always vertical align function args for readability unless it's < 2/3 args

Reply:
I think the official guide should be “format it vertically or as needed in order for each line to take less than 100 characters”, and after that we can just go depending on what looks better.